docs/blog/newsletter-march-2019.rst

```diff
@@ -5,21 +5,23 @@
 ######################################
 
-Hi everyone! Beth here, with your March newsletter. I know February is a short month, but it hardly feels like it's been a month since the last one.
+Hi everyone! Beth here, with your March newsletter. How on earth is February over already?
 
 We've got lots of conference news to share with you. Potential speakers, you'll be excited to hear that the `Prague Call for Proposals </conf/prague/2019/news/cfp-open/>`_ is open. Get working on those submissions - they need to be in by midnight (CEST) on Tuesday 30 April.
 
-And there's even more going on in Europe: `a new conference in Vilnius, Lithuania </conf/vilnius/2019/news/init/>`_! It'll be two days, 2-3 June, and the CFP and ticket sales are opening soon.
+And there's even more going on in Europe: `a new conference in Vilnius, Lithuania </conf/vilnius/2019/news/init/>`_! It'll be two days, 2-3 June; the CFP and ticket sales are opening soon.
 
-On the other side of the pond, `Portland speakers have been announced </conf/portland/2019/news/TODO>`_. If you're considering buying tickets, do it sooner rather than later: they sell out earlier every year. Want to go but need to persuade someone senior that it's worthwhile? The "`convince your manager </conf/portland/2019/convince-your-manager/>`_" page might help.
+On the other side of the pond, `Portland speakers have been announced </conf/portland/2019/news/announcing-speakers/>`_. If you're considering buying tickets, do it soon so you don't miss out: they sell out earlier every year. Want to go but need to persuade someone senior that it's worthwhile? The "`convince your manager </conf/portland/2019/convince-your-manager/>`_" page might help.
 
-So that's that. On to what we've been talking about on Slack this month!
+You may also be interested to know that we've updated the `code of conduct </code-of-conduct/>`_. Some sections have been improved and others clarified - for full details, check out the `code of conduct blog post </blog/new-community-coc-2019/>`_.
+
+On to what we've been talking about on Slack this month!
 
 ------------------------------------------
 Roadmaps and documenting 'future features'
 ------------------------------------------
 
-As documentarians, we can come head-to-head with other stakeholders on what to include in our documentation. One contentious topic is forward-looking statements: some stakeholders might want to tell users that a feature is coming soon. But when we mention a future feature in docs, it can be a problem if that feature doesn't end up being delivered.
+As documentarians, we sometimes come head-to-head with other stakeholders on what to include in our documentation. One contentious topic is forward-looking statements: some stakeholders might want to tell users that a feature is coming soon. But when we mention a future feature in docs, it can be a problem if that feature doesn't end up being delivered.
 
 Here are some thoughts about how to approach this:
```

Comment:
So it looks like we're missing the list here?

Reply:
I don't quite understand your comment?

Reply:
Oh, I was looking at the wrong view - it had looked like the bulleted list that you have under "Here are some thoughts about how to approach this" was just absent. But it's there! 👍